cfme/tests/control/test_basic.py

```diff
@@ -457,3 +458,16 @@ def test_alert_profile_crud(alert_profile):
     with update(alert_profile):
         alert_profile.notes = "Modified!"
     alert_profile.delete()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(blockers=[BZ(1416311, forced_streams=["5.7"])])
+@pytest.mark.uncollectif(lambda alert_profile: alert_profile.TYPE == "Middleware Server" and
+    current_version() < "5.7")
+def test_alert_profile_assigning(alert_profile):
+    alert_profile.create()
+    if alert_profile.TYPE == "Server":
```

Comment:
Dima, we can't do it like this; the string has to go.
It's either
`if alert_profile.TYPE == ServerAlertProfile.TYPE`
or
`if type(alert_profile) is ServerAlertProfile` (which I would strongly prefer).